packages/components/src/components/Progress/ProgressItem.tsx

```diff
@@ -0,0 +1,232 @@
+           */
+          tasks?: never;
+          /**
+           * Manually set the overall status of the progress item. Valid statuses; 'notStarted', 'inProgress', 'success', or 'error'.
+           */
+          status: ProgressStatus;
+      };
+
+export type ProgressItemProps = {
+    /**
+     * Optional title for the progress item
+     */
+    title?: string;
+    /**
+     * Optional description to offer more details about the progress or process, giving context to the progress status.
+     */
+    description?: string;
+    /**
+     * Flag indicating whether a retry button should be shown, allowing users to attempt the task again if it fails.
+     */
+    showRetryButton?: boolean;
+    /**
+     * Flag indicating whether a button to copy error details to the clipboard should be shown, useful for error reporting and diagnostics.
+     */
+    showCopyTextButton?: boolean;
+    /**
+     * Callback function that is called when the copy text button is pressed, providing the error details of the failed task.
+     * @param message An object containing details of the task error.
+     */
+    onCopyTextButtonPress?: (message: ProgressTaskErrorDetails) => void;
+    /**
+     * Callback function that is invoked when the retry button is pressed, allowing the specific failed task to be retried.
+     * @param task The task object that failed and needs to be retried.
+     */
+    onRetryButtonPress?: (task: ProgressTask) => void;
+} & ProgressItemPropsOptions;
+
+export const ProgressItem = ({
+    title,
+    description,
+    showCopyTextButton,
+    showRetryButton,
+    status = "notStarted",
+    tasks = [],
+    onCopyTextButtonPress,
+    onRetryButtonPress,
+}: ProgressItemProps) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+
+    const token = useToken();
+    const styles = useStyles(themeStyles);
+
+    const taskCounter = tasks?.length;
+    const completedTaskCounter = tasks?.filter(task => task.status === "success").length;
+
+    const animationConfig = {
+        duration: token.timing.animation.slow,
+        create: {
+            type: LayoutAnimation.Types.easeInEaseOut,
+            property: LayoutAnimation.Properties.opacity,
+        },
+        update: {
+            type: LayoutAnimation.Types.easeInEaseOut,
+        },
+    };
+
+    const taskStatus = computeTaskStatus(tasks, status);
+    const taskHasError = taskStatus === "error";
+    const failedTask = tasks.find(task => task.status === "error");
+
+    const handleCopyTextButtonPress = () => {
+        if (failedTask?.errorDetails) {
+            onCopyTextButtonPress && onCopyTextButtonPress(failedTask?.errorDetails);
+        }
+    };
+
+    const handleRetryButtonPress = () => {
+        if (failedTask) {
+            onRetryButtonPress && onRetryButtonPress(failedTask);
+        }
+    };
+
+    const toggleExpand = () => {
+        setIsExpanded(!isExpanded);
+        LayoutAnimation.configureNext(animationConfig);
```

Comment:
I can't seem to get the animation to work on web. Is this a known issue?
You could potentially look into using React Native Reanimated for animations as well. I know we probably should use that instead of the normal native one.
Don't stress this if it becomes too hard to implement though. Just a suggestion

Reply:
This should be fixed now. Changed to react-native-reanimated